utils/frame/rpc/support/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "frame-support-rpc"
+name = "substrate-frame-rpc-support"
```

Comment:
I'm not a huge fan of names here, why is it both `substrate` and `frame` and why `rpc` is before `support`?

Reply:
because they depend on both substrate and frame (not only either, but both), taking either out implies they wouldn't need the other IMHO...

`rpc` before the key modules because I thinking of a generic pattern to describe the integration: `substrate-frame` to define the integration barriers) + `-rpc` signifying the feature upon which the integration is happening + `-pallet` to name the specific pallet this relates to.

But as said before, I consider this a temporary place for either of them and just a reminder, that we need to fix that API here to make the distinction clear and move it back.